main/ZgatewayRF.ino

```diff
@@ -94,7 +94,8 @@ void RFtoMQTT() {
       RFtoMQTTdiscovery(MQTTvalue);
 #  endif
       pub(subjectRFtoMQTT, RFdata);
-      Log.trace(F("Store val: %lu" CR), MQTTvalue);
+      // Casting "receivedSignal[o].value" to (unsigned long) because ArduinoLog doesn't support uint64_t for ESP's 
```

Comment:
I think we should propose a PR to the ArduinoLog repository, I can do it if you want.

Reply:
To be honest, I'm not really sure how to deal with printing uint64_t values. I haven't used them yet in other projects.

I know I'm only printing the partial value with by using my cast, but I'm not sure how to fix it in the ArduinoLog library.

Reply:
You are right, not so easy as the `print `function doesn't support long long.